docs/tutorials/extending_admin_ui/6_adding_a_custom_tag.md

```diff
@@ -0,0 +1,127 @@
+# Creating a custom tag
+
+eZ Platform's RichText Field contains some built-in elements you can use, such as an image or a table.
+You can also add custom tags that will enable you to add additional elements to RichText Fields.
+
+In this step you will add a custom tag which will enable you to embed YouTube videos in Fields.
+
+## Configure the custom tag
+
+First, create a file that will contain the configuration for your custom tags.
+Add file `custom_tags.yml` to `src/EzSystems/ExtendingTutorialBundle/Resources/config`:
+
+``` yaml hl_lines="5 10 12"
+system:
+    default:
+        fieldtypes:
+            ezrichtext:
+                custom_tags: [ezyoutube]
+
+ezrichtext:
+    custom_tags:
+        ezyoutube:
+            template: 'AcmeTutorialBundle:field_type/ezrichtext/custom_tag:ezyoutube.html.twig'
+            icon: '/bundles/ezplatformadminui/img/ez-icons.svg#twitter'
+            attributes:
+                title:
+                    type: 'string'
+                    required: true
+                    default_value: ''
+                video_url:
+                    type: 'string'
+                    required: true
+                width:
+                    type: 'number'
+                    required: true
+                    default_value: 640
+                height:
+                    type: 'number'
+                    required: true
+                    default_value: 360
+                autoplay:
+                    type: 'boolean'
+                    default_value: false
+                align:
+                    type: 'choice'
+                    required: false
+                    default_value: 'left'
+                    choices: ['left', 'center', 'right']
+```
+
+The configuration first lists all custom tags that you have in the configuration (line 5) - in this case `ezyoutube`.
+`ezyoutube` is then configured. Line 10 points to the template used to render the tag.
+Then attributes of the custom tag are listed. These attributes can be set when adding the tag to a RichText Field.
+
+## Create a template
+
+Next, create the template that is referred to in the configuration.
+In `src/EzSystems/ExtendingTutorialBundle/Resources/views/field_type/ezrichtext/custom_tag` add the following `ezyoutube.html.twig` file:
+
+``` twig
+<div{% if params.align is defined %} style="text-align: {{ params.align }};"{% endif %}>
+    <iframe
+        type="text/html"
+        width="{{ params.width }}"
+        height="{{ params.height }}"
+        src="{{ params.video_url|replace({'https://youtu.be/' : 'https://www.youtube.com/embed/'}) }}?autoplay={{ params.autoplay == 'true' ? 1 : 0 }}"
```

Comment:
I would add a tip that url for video should be taken from `share` button on youtube

Reply:
Explained at the end of the page.